<?xml version="1.0"?>

<launch>
    <arg name="respawn_node" default="false"/>
    <arg name="node_name" default="test_publish" />
    <arg name="local_host" default="true"/>
    <arg name="params_path" default="$(find ros_kafka_connector)/config/params.yaml"/>

    <node pkg="ros_kafka_connector" type="test_publish.py" name="$(arg node_name)" output="screen" respawn="$(arg respawn_node)">
        <rosparam command="load" file="$(arg params_path)"  subst_value="true"/>
    </node>

</launch>
